src/FunctionApps/python/IntakePipeline/__init__.py

```diff
@@ -7,8 +8,18 @@
 from IntakePipeline.fhir import read_fhir_bundles, upload_bundle_to_fhir_server
 
 
+def get_required_config(varname: str) -> str:
+    """Grab a required config val and throw an exception if it's not present"""
+    if varname not in os.environ:
+        raise Exception(f"Config value {varname} not found in the environment")
+    return os.environ[varname]
+
+
 def run_pipeline():
-    for bundle in read_fhir_bundles():
+    for bundle in read_fhir_bundles(
+        get_required_config("INTAKE_CONTAINER_URL"),
+        get_required_config("INTAKE_CONTAINER_PREFIX"),
```

Comment:
Is it worth just storing these two values in strings above the for loop, so that we don't have to make a pair of function calls with every bundle that passes through? e.g.

```
intake_container_url = get_required_config()
intake_container_prefix = get_required_config()
for bundle in read_fhir_bundles(intake_container_url, intake_container_prefix):
    pass
```

I stored the salt hash that way.

Reply:
:shrug: env var reading is quick